packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -465,7 +465,7 @@ const LivechatTagsListSchema = {
 
 export const isLivechatTagsListProps = ajv.compile<LivechatTagsListProps>(LivechatTagsListSchema);
 
-type LivechatDepartmentProps = PaginatedRequest<{
+export type LivechatDepartmentProps = PaginatedRequest<{
```

Comment:
`question` Is this export really necessary?

```suggestion
type LivechatDepartmentProps = PaginatedRequest<{
```

Reply:
Nope, left overs from unused code, removing